pkg/prometheus/server/rules_test.go

```diff
@@ -54,9 +51,8 @@ func shouldErrorOnTooLargeRuleFile(t *testing.T) {
 	ruleFiles["my-rule-file"] = strings.Repeat("a", v1.MaxSecretSize+1)
 
 	_, err := makeRulesConfigMaps(&monitoringv1.Prometheus{ObjectMeta: metav1.ObjectMeta{Name: "test"}}, ruleFiles)
-	if err == nil || err.Error() != expectedError {
-		t.Fatalf("expected makeRulesConfigMaps to return error '%v' but got '%v'", expectedError, err)
-	}
+	require.Error(t, err)
+	require.Equal(t, expectedError, err.Error(), "expected makeRulesConfigMaps to return error '%v' but got '%v'", expectedError, err)
```

Comment:
```suggestion
	require.Equal(t, expectedError, err.Error(), "expected makeRulesConfigMaps to return error '%v' but got '%v'", expectedError, err)
```

If err is nil, the `require.Equal(...)` will catch it already, right?